views/signedup.tmpl

```diff
@@ -0,0 +1,100 @@
+{{define "title"}}{{.PageContext.ShortName}} | Sign Up{{end}}
+
+{{define "open-graph"}}
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@{{.PageContext.MainTwitter}}">
+<meta name="twitter:creator" content="@{{.PageContext.MainTwitter}}">
+<meta property="og:type" content="website">
+<meta property="og:locale" content="en_GB">
+<meta property="og:title" content="{{.PageContext.LongName}} | Sign Up">
+<meta property="og:description" content="{{.PageContext.SiteDescription}}">
+<meta property="og:image" content="{{.PageContext.FullURL}}{{url "/images/open_graph-general.jpg"}}">
+<meta property="og:url" content="{{.PageContext.FullURL}}{{url "/signup/"}}">
+{{end}}
+
+{{define "content"}}
+{{with .PageData}}
+
+<div class="container-fluid banner-2">
+  <div class="container container-padded">
+    <div class="row align-items-center text-center">
+      <div class="col">
+        <h1 class="display-3">On Air, Online, On Demand</h1>
+        <h2 class="display-4">Broadcasting since 1968.</h2>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container-fluid container-padded bg-primary">
+	<div class="container container-padded">
```

Comment:
inconsistent use of tabs and spaces, currently I think we are trying to use 2 spaces